src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -109,6 +108,8 @@ public void acquireLease() throws ConnectException {
             ),
             e
         );
+      } finally {
```

Comment:
Hey @avocader  moving to finally will close the reader/writer even when the lease is successfully acquired.  I do not see that in the old code.  Are we sure this is the right fix?

Reply:
I deleted `close()` call, so not relevant anymore.